### PR TITLE
Show the description column on environment tab

### PR DESCRIPTION
**Pull request by vinay033**
Signed-off-by: Vinay Sharma <vsharma@chef.io>

### :nut_and_bolt: Description: What code changed, and why?
we need to add a description column on the infra proxy environment tab.

needDesc

### :chains: Related Resources
https://github.com/chef/automate/issues/4721
### :+1: Definition of Done
I have added changes to show the description column on the environment tab
### :information_source: Note
This PR is not containing the new design CSS, it depends on PR: https://github.com/chef/automate/pull/4673
after merging the dependent PR this page show like same as the attached screenshot.
### :athletic_shoe: How to Build and Test the Change
if you have sample data for the infra servers than
go to >> infrastructure tab >> click on Chef server on left side navigation bar >> Organizations >> Environments tab >> see the list of environments.
### :white_check_mark: Checklist

- [ ] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] Tests added/updated?
- [ ] Docs added/updated?
- [ ] All commits have been signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### :camera: Screenshots, if applicable
Description

### CODE REVIEW

**Comment by shaik80** [comment]
Screenshot 2021-02-12 at 5 40 37 PM
Not able to view description data

**Comment by vinay033** [comment]
> Screenshot 2021-02-12 at 5 40 37 PM
> 
> Not able to view description data tried to rebuild infra-proxy-service getting the same result to me

have you checked on manage, if manage has a description available for a particular environment then the same data reflecting on automate.

File: components/automate-ui/src/app/modules/infra-proxy/environments/environments.component.html [hunk 1]
```diff
@@ -4,15 +4,25 @@
...
+          <chef-th class="no_border_th"></chef-th>
         </chef-tr>
       </chef-thead>
       <chef-tbody>
         <chef-tr *ngFor="let environment of environments">
           <chef-td>
             <a [routerLink]="['/infrastructure','chef-servers', serverId, 'organizations', orgId, 'environments', environment.name]">{{ environment.name }}</a>
           </chef-td>
+          <chef-td>
+            {{ environment.description }}
```
Comment (kalroy) [inline components/automate-ui/src/app/modules/infra-proxy/environments/environments.component.html:21]: With long description the text is getting wrapped although we have enough space to the right. 
image
Is this how the text should look like?

**Review by kalroy — CHANGES_REQUESTED** [review]

**Review by shaik80 — APPROVED** [review]

**Comment by shaik80** [comment]
**👟 How to Build and Test the Change**
Need to rebuild gateway, UI, infra-proxy before testing UI changes

**Pull request merged**